### PR TITLE
[FIX] Fix translation fr

### PR DESCRIPTION
**Pull request by rimbendhaou**

### CODE REVIEW

File: project_event/i18n/fr.po [hunk 1]
```diff
@@ -687,7 +686,7 @@ msgstr "Horaire Prévu"
 #: model:ir.ui.view,arch_db:project_event.res_partner_sector_for_view
 #: model:ir.ui.view,arch_db:project_event.view_project_project_search
 msgid "Faculty Sectors"
-msgstr "Secteur de Faculté"
+msgstr "Secteur"
```
Comment (jananjoy) [inline project_event/i18n/fr.po:689]: Not the corresponding translation of the english text

File: project_event/i18n/fr.po [hunk 2]
```diff
@@ -191,13 +190,13 @@ msgid "Activity:"
...
 #. module: project_event
-#: code:addons/project_event/models/project_project.py:182
+#: code:addons/project_event/models/project_project.py:184
 #, python-format
 msgid "Activity: "
 msgstr "Activité: "
 
 #. module: project_event
-#: code:addons/project_event/models/project_task.py:882
+#: code:addons/project_event/models/project_task.py:891
```
Comment (jananjoy) [inline project_event/i18n/fr.po:199]: Two entry for activity for two files. can we keep one entry referencing the two files

File: project_event/i18n/fr.po [hunk 3]
```diff
@@ -459,8 +458,8 @@ msgid "Display Name"
 msgstr "Nom à afficher"
 
 #. module: project_event
-#: code:addons/project_event/models/project_project.py:212
-#: code:addons/project_event/models/project_task.py:956
+#: code:addons/project_event/models/project_project.py:214
```
Comment (jananjoy) [inline project_event/i18n/fr.po:461]: This does not match with the references

File: project_event/i18n/fr.po [hunk 4]
```diff
@@ -540,7 +539,7 @@ msgid "Equipment"
 msgstr "Équipement"
 
 #. module: project_event
-#: code:addons/project_event/report/project_task_activity_report.py:65
+#: code:addons/project_event/report/project_task_activity_report.py:66
 #, python-format
 msgid "Equipment: <br/>"
 msgstr "Equipment: <br/>"
```
Comment (jananjoy) [inline project_event/i18n/fr.po]: Is it correct translation in French. Plz refer the spec.

File: project_event/i18n/fr.po [hunk 5]
```diff
@@ -882,8 +881,8 @@ msgid "Phone:"
 msgstr "Téléphone :"
 
 #. module: project_event
-#: code:addons/project_event/models/project_project.py:212
-#: code:addons/project_event/models/project_task.py:956
+#: code:addons/project_event/models/project_project.py:214
+#: code:addons/project_event/models/project_task.py:965
 #, python-format
 msgid "Please Confirm your reservation.<br>"
```
Comment (jananjoy) [inline project_event/i18n/fr.po]: Change to Please 'confirm' your reservation.

File: project_event/i18n/fr.po [hunk 6]
```diff
@@ -1060,7 +1060,7 @@ msgid "Room"
 msgstr "Salle"
 
 #. module: project_event
-#: code:addons/project_event/report/project_task_activity_report.py:64
+#: code:addons/project_event/report/project_task_activity_report.py:65
 #, python-format
 msgid "Room: "
 msgstr "Salle:"
```
Comment (jananjoy) [inline project_event/i18n/fr.po]: Space missing  ' ' in the transtation

File: project_event/i18n/fr.po [hunk 7]
```diff
@@ -1256,7 +1257,7 @@ msgid "Tasks to Edit"
 msgstr "Tâches à modifier"
 
 #. module: project_event
-#: code:addons/project_event/models/project_task.py:883
+#: code:addons/project_event/models/project_task.py:892
 #, python-format
 msgid "Tasks: <br>"
 msgstr "Tâches: <br>"
```
Comment (jananjoy) [inline project_event/i18n/fr.po:1263]: code:addons/project_event/models/project_task.py:892
#: code:addons/project_event/models/project_task.py:899
Refer to the same msgid "Tasks: <br>" . Is it possible to add single entry for both reference(both in same file)

File: project_event/i18n/fr.po [hunk 8]
```diff
@@ -1282,8 +1283,8 @@ msgid "Templates Events"
 msgstr "Modèles d'événements"
 
 #. module: project_event
-#: code:addons/project_event/models/project_project.py:211
-#: code:addons/project_event/models/project_task.py:955
+#: code:addons/project_event/models/project_project.py:213
+#: code:addons/project_event/models/project_task.py:964
 #, python-format
 msgid "The Following resources are already booked:<br>"
```
Comment (jananjoy) [inline project_event/i18n/fr.po]: 'F' should be small in Following

File: project_event/i18n/fr.po [hunk 9]
```diff
@@ -1299,8 +1300,8 @@ msgstr ""
 "catégorie par défaut:"
 
 #. module: project_event
-#: code:addons/project_event/models/project_project.py:168
-#: code:addons/project_event/models/project_task.py:865
+#: code:addons/project_event/models/project_project.py:170
+#: code:addons/project_event/models/project_task.py:874
 #, python-format
 msgid ""
 "The following is Optional and                         appears as "
```
Comment (jananjoy) [inline project_event/i18n/fr.po]: Optional with small 'o'

File: project_event/i18n/fr.po [hunk 10]
```diff
@@ -1423,13 +1430,13 @@ msgid "res.partner.sector"
 msgstr "res.partner.sector"
 
 #. module: project_event
-#: code:addons/project_event/models/project_task.py:380
+#: code:addons/project_event/models/project_task.py:381
 #, python-format
 msgid "this resource is not bookable"
```
Comment (jananjoy) [inline project_event/i18n/fr.po:1435]: Change this to This (Sentence case)

File: project_event/i18n/fr.po [hunk 11]
```diff
@@ -1423,13 +1430,13 @@ msgid "res.partner.sector"
...
 msgstr "Cette ressource n'est pas réservable"
 
 #. module: project_event
-#: code:addons/project_event/models/project_task.py:382
+#: code:addons/project_event/models/project_task.py:383
 #, python-format
 msgid "this room is not bookable"
```
Comment (jananjoy) [inline project_event/i18n/fr.po:1441]: Change this to This

File: project_resource_calendar/i18n/fr.po [hunk 12]
```diff
@@ -7,66 +7,50 @@ msgid ""
...
+"X-Generator: Poedit 1.8.7.1\n"
 
 #. module: project_resource_calendar
-#: code:addons/project_resource_calendar/models/calendar_event.py:110
-#: code:addons/project_resource_calendar/models/calendar_event.py:116
+#: code:addons/project_resource_calendar/models/calendar_event.py:134
+#: code:addons/project_resource_calendar/models/calendar_event.py:140
 #, python-format
 msgid " Time(s)"
 msgstr " fois "
```
Comment (jananjoy) [inline project_resource_calendar/i18n/fr.po]: Extra Space

File: project_resource_calendar/i18n/fr.po [hunk 13]
```diff
@@ -126,8 +110,16 @@ msgstr "Capacité"
...
-msgstr "Vérifier si la ressource peut être réservée plusieurs fois aux même temps."
+msgid ""
+"Check if this resource can be booked in more than one meeting or event at "
+"the same time."
+msgstr "Vérifier si la ressource peut être réservée concomitamment."
+
+#. module: project_resource_calendar
+#: code:addons/project_resource_calendar/models/ir_actions_report.py:26
+#, python-format
+msgid "Choose Your Document Layout"
```
Comment (jananjoy) [inline project_resource_calendar/i18n/fr.po:121]: Better to  change Choose your document layout

File: project_resource_calendar/i18n/fr.po [hunk 14]
```diff
@@ -766,7 +742,19 @@ msgstr "salle"
 #. module: project_resource_calendar
 #: model:ir.ui.view,arch_db:project_resource_calendar.resource_room_type_tree
 msgid "room_type"
-msgstr "room_type"
+msgstr "Type de salle"
+
+#. module: project_resource_calendar
+#: code:addons/project_resource_calendar/models/calendar_event.py:206
+#, python-format
+msgid "this resource is not bookable"
```
Comment (jananjoy) [inline project_resource_calendar/i18n/fr.po:750]: change this to This

File: project_event/i18n/fr.po (hunk 1 above)
Comment (rimbendhaou) [reply project_event/i18n/fr.po:689]: It correspond to the functional spec

File: project_event/i18n/fr.po (hunk 10 above)
Comment (rimbendhaou) [reply project_event/i18n/fr.po:1435]: It comes after : so i have to change the fr to lower case

File: project_resource_calendar/i18n/fr.po (hunk 13 above)
Comment (rimbendhaou) [reply project_resource_calendar/i18n/fr.po:121]: It was written like this in odoo

**Review by jananjoy — APPROVED** [review]
Look good

**Pull request merged**